pkg/f1/f1_root_cmd.go

```diff
@@ -84,9 +88,22 @@ func writeProfiles() {
 }
 
 func ExecuteWithArgs(args []string) error {
+	configureHttpDefaults()
 	rootCmd := buildRootCmd()
 	rootCmd.SetArgs(args)
 	err := rootCmd.Execute()
 	writeProfiles()
 	return err
 }
+
+func configureHttpDefaults() {
+	runtime.GOMAXPROCS(48)
```

Comment:
I'm not sure this one is needed - think we probably added it in a panic when we didn't realise what we were doing.